### DISCUSSION TITLE
2024/12/05/%E6%95%B0%E7%94%B5%E7%AC%94%E8%AE%B0-2/

### DISCUSSION THREAD

**Discussion by giscus** [TheUHO/TheUHO.github.io#7] [Announcements]
# 2024/12/05/%E6%95%B0%E7%94%B5%E7%AC%94%E8%AE%B0-2/

杂质半导体与PN结  N型半导体: 自由电子为多子，空穴为少子，以电子导电为主 P型半导体: 自由电子为少子，空穴为多子，以空穴导电为主 杂质半导体中，多子浓度由杂质含量决定，少子浓度主要由温度决定 PN结的伏安特性:  正向导通: i=Is(ev/VT−1)i = I_s(e^{v/V_{T}}-1)i=Is​(ev/VT​−1) IsI_sIs​为反向饱和电流，VTV_TVT​为电压当量，室温

https://wwwwwwangzx.github.io/2024/12/05/%E6%95%B0%E7%94%B5%E7%AC%94%E8%AE%B0-2/

**Comment by TheUHO** [comment]
tql%%%
